sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_counts_v1/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: FxA DB Counts Monitoring
+description: |-
+  Simple aggregation of counts of records in the FxA DB tables.
+  Enables to identify trends within accounts data. E.g. "How many
+  inactive accounts are there?"
+  Note: because its source tables are overwritten daily, query used to
+  populate this table is not idempotent so this table should not be backfilled.
```

Comment:
As it is, if someone re-runs a historical DAG run then historical data would be incorrectly overwritten with current data (which is highly probable since other normal daily-incremental ETLs exist in the `bqetl_accounts_derived` DAG).

I'd suggest using BigQuery's time travel feature in this ETL to select the data as it existed at `data_interval_end` so that any DAG run within the last ~7 days could potentially be backfilled, but trying to backfill further than time travel allows will fail outright rather than incorrectly overwriting data ([example](https://github.com/mozilla/private-bigquery-etl/tree/main/sql/moz-fx-data-bq-people/workday_derived/monthly_active_employees_v1)).

Reply:
Cool, thanks for this idea!